File: samples/VB/18-SizeDefinition/Form1.vb
﻿Imports WinForms.Ribbon

Namespace _18_SizeDefinition
    Partial Public Class Form1
        Inherits Form
        Private _ribbonItems As RibbonItems
        Public Sub New()
            InitializeComponent()
            _ribbonItems = New RibbonItems(_ribbon, True)
            _ribbonItems.Init()
        End Sub
    End Class
End Namespace

File: samples/CS/18-SizeDefinition/Form1.cs
using System;
using System.Windows.Forms;

using WinForms.Ribbon;

namespace _18_SizeDefinition
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon, true);
            _ribbonItems.Init();
        }
    }
}
